src/styles/main.scss

```
@@ -1,3 +1,23 @@
-body {
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  
+  max-width: 984px;
+  padding: 0 20px;
 }
+
+@import "./blocks/header.scss";
+@import "./blocks/cards-section.scss";
+@import "./blocks/card.scss";
+@import "./blocks/stars.scss";
+@import "./blocks/button.scss";
```

don't need on each file import variables and extends. You can here import them once 

```suggestion
@import "./utils/variables";
@import "./utils/extends";
@import "./blocks/header.scss";
@import "./blocks/cards-section.scss";
@import "./blocks/card.scss";
@import "./blocks/stars.scss";
@import "./blocks/button.scss";
```